mteb/tasks/Retrieval/eng/LEMBNarrativeQARetrieval.py

```diff
@@ -0,0 +1,67 @@
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class LEMBNarrativeQARetrieval(AbsTaskRetrieval):
+    _EVAL_SPLIT = "test"
+
+    metadata = TaskMetadata(
+        name="LEMBNarrativeQARetrieval",
+        dataset={
+            "path": "dwzhu/LongEmbed",
+            "revision": "6e346642246bfb4928c560ee08640dc84d074e8c",
+            "name": "narrativeqa",
+        },
+        reference="https://huggingface.co/datasets/dwzhu/LongEmbed",
+        description=("narrativeqa subset of dwzhu/LongEmbed dataset."),
+        type="Retrieval",
+        category="s2p",
+        eval_splits=[_EVAL_SPLIT],
+        eval_langs=["eng-Latn"],
+        main_score="ndcg_at_10",
+        date=None,
```

Comment:
when the text was created (from, to) or best guess.

Reply:
take Narrative for example, do you mean when the original dataset is created (which is back in 2020), or when it is curated for retrieval (which could be today)

Reply:
Text creation time, so if it e.g. includes books written from 1800-2000. These can be approximate guesses. Given that it was created in 2020, that at least set the upper bound.

Reply:
I see, thank you!